client/api.go

```diff
@@ -94,6 +95,8 @@ type API struct {
 	BankAccounts *bankaccount.Client
 	// BillingPortalSessions is the client used to invoke /billing_portal/sessions APIs.
 	BillingPortalSessions *billingportalsession.Client
+	// BillingPortalSessions is the client used to invoke /billing_portal/configurations APIs.
+	BillingPortalConfigurations *billingportalconfiguration.Client
```

Comment:
Something is missing. I am convinced I always had to do 3 separate things in that file

Reply:
I found 2 -- this one and the one on line 252. If you count the docstring then that's 3?

Reply:
Hmmm maybe I missed one, I was sure I saw only 2 modifications in that file instead of 3 (like you have now) but I might have missed it